projects/igniteui-angular/src/lib/action-strip/action-strip.component.ts

```diff
@@ -279,6 +279,9 @@ export class IgxActionStripComponent implements IgxActionStripToken, AfterConten
      * ```
      */
     public show(context?: any): void {
+        if(!this._originalParent) {
```

Comment:
Hm, if every workflow goes through `show` we might even move the caching of the original parent here `ngAfterViewInit` where it was originally, so both cache and return are on the show/hide pair.

Anyway, I'm assuming this fixes the disappearing strip if it inits in some case as detached, but still is attached before showing.